docs/.vuepress/components/meta-data.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <header class="meta-data" v-if="!isHomePage">
+    <span class="last-updated">
+      Last updated: <time :datatime="prettyDate">{{prettyDate}}</time>
+    </span>
+    <a
```

Comment:
Maybe nice to make this a `rel="noopener" target="_blank"` so you can still keep the Playbook open?